src/sentry/models/exporteddata.py

```diff
@@ -33,12 +33,11 @@ class ExportedData(Model):
 
     organization = FlexibleForeignKey("sentry.Organization")
     user = FlexibleForeignKey(settings.AUTH_USER_MODEL)
+    file = FlexibleForeignKey("sentry.File", null=True)
```

Comment:
The `File` table looks like it has ~1B rows. I assume this means it's a fairly busy read/write table, and so we should be careful about adding a db level FK constraint to it. IMO it might make sense to just skip the constraint entirely to be safe. 

To do so, we can set `db_constraint=False` to this definition here and regenerate the migration. This allows us to still have an FK in Django so that we can treat it like a relation, but we don't get the db level consistency guarantees, which seems like an ok tradeoff.

cc @mattrobenolt in case you know something I don't about this table.

Reply:
Yeah, if you can live without the constraint, that's probably for the better.

Reply:
Regenerated and modified in the latest commit. SQL is as follows:
```sql
BEGIN;
--
-- Remove field storage_url from exporteddata
--
ALTER TABLE "sentry_exporteddata" DROP COLUMN "storage_url" CASCADE;
--
-- Add field file to exporteddata
--
ALTER TABLE "sentry_exporteddata" ADD COLUMN "file_id" bigint NULL;
CREATE INDEX "sentry_exporteddata_file_id_a9a0b802" ON "sentry_exporteddata" ("file_id");
COMMIT;
```